database/seeds/UsersTableSeeder.php

```diff
@@ -9,25 +9,27 @@ class UsersTableSeeder extends Seeder
 {
     /**
      * Run the database seeds.
+     *
+     * @return void
      */
     public function run()
     {
         $admins = [
-                [
-                  'name' => 'Diogo Couto',
-                  'email' => 'diogo2couto@gmail.com',
-                  'github' => 'djcouto',
-                ],
+            [
+                'name' => 'Diogo Couto',
+                'email' => 'diogo2couto@gmail.com',
+                'github' => 'djcouto',
+            ],
         ];
 
         $adminRole = Role::whereType(Role::ROLE_ADMINISTRATOR)->first();
         foreach ($admins as $user) {
             User::create([
-            'name' => $user['name'],
-            'email' => $user['email'],
-            'github' => $user['github'],
-            'password' => bcrypt(Str::random()),
-            'role_id' => $adminRole->id,
+                'name' => $user['name'],
+                'email' => $user['email'],
+                'github' => $user['github'],
+                'password' => bcrypt(Str::random()),
+                'role_id' => $adminRole->id,
           ]);
```

Comment:
Indentation is not ok here.

Reply:
Third attempt, don't know which is the right indentation :joy:

Reply:
4 spaces :D